package.json

```diff
@@ -22,6 +22,7 @@
     "url": "https://github.com/badges/shields"
   },
   "dependencies": {
+    "anafanafo": "^0.1.0",
```

Comment:
This requirement should be in `gh-badges/package.json` instead of here

Reply:
Oops, thanks. That was a merge error.